apps/developer-hub/next.config.js

```diff
@@ -0,0 +1,77 @@
+import createBundleAnalyzer from "@next/bundle-analyzer";
```

Comment:
Nice!  Note that I'm working on moving other apps to using turbopack; I'm not sure how well this will work with turbopack, but in the meantime I think it would be great to roll it out across all our apps